src/lib.rs

```diff
@@ -39,7 +39,8 @@
 #[cfg(feature = "std")]
 extern crate std;
 
-extern crate alloc;
+pub extern crate alloc;
+pub extern crate spin;
```

Comment:
Let's not commit to these as part of the public API.
```suggestion
#[doc(hidden)]
pub extern crate alloc;
#[doc(hidden)]
pub extern crate spin;
```

Reply:
Fixed.